src/frontend/ts/types/urbitChess.ts

```diff
@@ -133,6 +141,7 @@ export interface PositionUpdate extends ChessUpdate {
   gameID: GameID
   position: FENPosition
   specialDrawAvailable: boolean
+  move: Move | null
```

Comment:
Sorry, it's been a while - when is it possible for this to be `null`?

Reply:
I'm sure there was a good reason for this that wasn't "the app doesn't load unless this is allowed to be null", but I tried removing `| null` and turns out the app doesn't load unless this is allowed to be null.

Every time I've had this "the whole app crashes/doesn't load" error it's because it's trying to display something that isn't there. In this case the app is doing something as soon as I open it that causes this error.

Seems like when you open the frontend it subscribes to the `/updates` wire, then the `on-watch` endpoint sends over a `%position` update for every move that's been made in the game. If no moves have been made in that game, then the only update being sent has `null` where a move should be.

This is the relevant code from `on-watch`:
```
=/  cards  ^-  (list card)
   %+  turn
     moves.game.game-state
   |=  move=[move=chess-move fen=chess-fen san=chess-san]
   :*  %give  %fact   ~[/game/(scot %da u.game-id)/updates]
       %chess-update  !>([%position u.game-id fen.move special-draw-available.game-state san.move])
   ==
```

I've tried a couple of variations of "if `moves` is null then don't send anything", which cause issues on the frontend or backend. Could be that something has to give: we either *have to deal with* the value (with `| null`) or we *have to deal with* the whole game/cards/wire behaviour.

Reply:
That makes sense - letting it be `null` is definitely the easier solution. I think @rovmug-ticfyn has something similar in his PR for when someone tries to undo the first move.